schema/definitions.json

```diff
@@ -67,20 +67,51 @@
       "type": "object",
       "properties": {
         "url": { "$ref": "#/$defs/url" },
+        "status": {
+          "type": "string",
+          "enum": [
+            "Candidate Recommendation Draft",
+            "Candidate Recommendation Snapshot",
+            "Draft Note",
+            "Draft Registry",
+            "First Public Working Draft",
+            "Note",
+            "Proposed Recommendation",
+            "Recommendation",
+            "Retired",
```

Comment:
Maybe we should align "Retired" with the W3C Process term, [Discontinued Draft](https://www.w3.org/2021/Process-20211102/#discontinuedREC)

Reply:
I had kept "Retired", and am tempted to keep it, because that is what we use on /TR pages and the status that the W3C API returns:
https://www.w3.org/standards/history/encoding

Reply:
right - but my recollection is that I coined that term "Retired" out of thin air some 20 years ago in the context of modeling TR in RDF, and that trickled down into the 2008 Web site redesign and later on in the W3C API. Now that we have a better consensus-based name, I don't think there is actual value in sticking to that accident of history :)